src/renderer/features/multisig-wallet/index.tsx

```diff
@@ -75,10 +75,15 @@ accountSDK(multisigWalletFeature, {
         })
         .concat(children);
     }
+
+    if (accountUtils.isFlexibleProxiedAccount(account)) {
+      return accounts.filter(a => a.accountId === account.proxyAccountId).concat(children);
```

Comment:
can you also check walletId? 
```suggestion
      return accounts.filter(a => a.accountId === account.proxyAccountId && a.walletId === account.walletId).concat(children);
```